src/zenml/deployers/exceptions.py

```diff
@@ -0,0 +1,76 @@
+#  Copyright (c) ZenML GmbH 2025. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Base class for all ZenML deployers."""
+
+from typing import (
+    TYPE_CHECKING,
+)
+
+from zenml.exceptions import EntityExistsError
+from zenml.logger import get_logger
+
+if TYPE_CHECKING:
+    pass
+
+logger = get_logger(__name__)
+
+DEFAULT_DEPLOYMENT_LCM_TIMEOUT = 300
```

Comment:
That same constant also exists in the base deployer file, with a different value